index.js

```diff
@@ -1,10 +1,12 @@
+const imgUrl = 'http://xavid.us/img/';
```

Comment:
I think this will need to be http://xavid.us/quest/img/ to work with the images checked into git, which does seem like the best approach.

Reply:
Done.